instrumentation/opentelemetry-instrumentation-dbapi/src/opentelemetry/instrumentation/dbapi/__init__.py

```diff
@@ -211,6 +211,7 @@ def __init__(
         connection_attributes=None,
         version: str = "",
         tracer_provider: typing.Optional[TracerProvider] = None,
+        capture_parameters: bool = False,
```

Comment:
How can this be configured by the user? As well as the instrumentations that depend on this?

Reply:
Yeah, I did not make this configurable for the user.  I initially though I was going to but then came across this PR that removes the capturing of "db.statement.parameters" for AsyncPGInstrumentor
https://github.com/open-telemetry/opentelemetry-python/pull/854 Since the approach was accepted I thought it would be fine for DBAPI too.

Would it be better to not have the constructor param and instead introduce a user settable property? 

Thanks!

Reply:
Having users be able to configure it is fine. The reason why the `AsyncPGInstumentor` works is because the configuration property is exposed to the users (because they call `AsyncPGInstrumentor` constructor to instrument`). However for this case it is in `DatabaseApiIntegration`, which users do not interact with (for dbapi, they use `trace_integration(...)` or for something like mysql, they use MySQLInstrumentor()`.